chromadb/test/property/test_persist.py

```diff
@@ -231,14 +233,17 @@ def on_state_change(self, new_state: str) -> None:
             self.min_state_changes_left_before_persisting -= 1
 
     def teardown(self) -> None:
-        self.api.reset()
+        self.client.reset()
 
 
 def test_persist_embeddings_state(
     caplog: pytest.LogCaptureFixture, settings: Settings
 ) -> None:
     caplog.set_level(logging.ERROR)
-    api = chromadb.Client(settings)
+    client = chromadb.Client(settings)
     run_state_machine_as_test(
-        lambda: PersistEmbeddingsStateMachine(settings=settings, api=api),
+        lambda: PersistEmbeddingsStateMachine(settings=settings, client=client),
```

Comment:
is this a merge regress or did you mean to make this change?

Reply:
Not sure what this is referring to.